### PR TITLE
Readthedocs and deps fixes

### LINKED ISSUE #4180: Fix readthedocs

### PR DESCRIPTION
**Pull request by eccabay**
Closes #4180

### CODE REVIEW

**Comment by machineFL** [comment]
No linked issues found. Please add the corresponding issues in the pull request description. <br/> [Use GitHub automation to close the issue when a PR is merged](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)

File: .github/workflows/latest_dependency_checker.yaml [hunk 1]
```diff
@@ -32,4 +32,4 @@ jobs:
           delete-branch: true
           base: main
           assignees: machineFL
-          reviewers: machineAYX, jeremyliweishih, bchen1116, chukarsten, ParthivNaresh, eccabay, christopherbunn, fjlanasa, MichaelFu512, tamargrey
+          reviewers: jeremyliweishih, bchen1116, chukarsten, ParthivNaresh, eccabay, christopherbunn, fjlanasa, tamargrey
```
Comment (gsheni) [inline .github/workflows/latest_dependency_checker.yaml:35]:
```suggestion
          reviewers: jeremyliweishih, bchen1116, chukarsten, ParthivNaresh, eccabay, christopherbunn, fjlanasa
```

**Review by jeremyliweishih — APPROVED** [review]
👍

**Review by chukarsten — APPROVED** [review]
Nice job team!

**Pull request merged**